common/utils.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import "fmt"
+
+func Prompt(text, defaultValue string) string {
```

Comment:
Consider this:
https://github.com/c-bata/go-prompt
https://github.com/stromland/cobra-prompt

Reply:
I'm not sure that we need to use something complex to perform just one prompt.